src/libraries/System.Diagnostics.DiagnosticSource/tests/ActivitySourceTests.cs

```diff
@@ -881,6 +881,98 @@ public void RestoreOriginalParentTest()
             }).Dispose();
         }
 
+
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void TestActivityCreate()
```

Comment:
It would be nice to have at least one test case for each of the scenarios the API is intended to enable, ie 
(1) add tags/links/baggage after Create() and before Start()
(2) create a child Activity that has W3C format even when global format is Hierarchical

Reply:
I'll add these tests